x-pack/plugin/enrich/qa/rest-with-security/src/test/java/org/elasticsearch/xpack/enrich/EnrichSecurityIT.java

```diff
@@ -29,4 +33,14 @@ protected Settings restAdminSettings() {
             .put(ThreadContext.PREFIX + ".Authorization", token)
             .build();
     }
+
+    public void testImmutablePolicy() throws IOException {
```

Comment:
I think this test should be added to `CommonEnrichRestTestCase` instead of here?
The outcome should be the same whether security is enabled or not?

Reply:
lol i cant believe i put it there... blame my ailing mind.